src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -192,14 +192,14 @@ function IOURequestStepConfirmation({
 
             // If we have a receipt let's start the split bill by creating only the action, the transaction, and the group DM if needed
             if (iouType === CONST.IOU.TYPE.SPLIT && receiptFile) {
-                const existingSplitChatReportID = CONST.REGEX.NUMBER.test(reportID) ? reportID : '';
+                const existingSplitChatReportID = CONST.REGEX.NUMBER.test(report.reportID) ? reportID : '';
```

Comment:
This seems to be an existing bug. `reportID` is a reference to the reportID coming from the URL route, and that will always be a number. This changes the code so that it's testing if the `reportID` belongs to an optimistic report or not, which I think is what is intended.